.github/workflows/deploy.yml

```
@@ -1,38 +1,46 @@
+    branches:
+      - main
+    paths:
+      - 'apps/server/**'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)